### DISCUSSION TITLE
How to capture the full screen directly?

### DISCUSSION THREAD

**Discussion by XxRiKxX** [flameshot-org/flameshot#3193] [Q&A]
I am on a windows 10 vm and the ptrsc key asks to select the region when screenshoting. I want to directly capture it and save it to the path provided without any extra steps

**Comment by mmahmoudian** [comment]
In windows you cannot do it with only printscreen because we do not provide CLI for windows yet, but you can use <kbd>Ctrl</kbd><kbd>a</kbd> to do "select all" and then normal stuff, like <kbd>Ctrl</kbd><kbd>c</kbd> to copy, <kbd>Ctrl</kbd><kbd>s</kbd> to save to file and etc.

You can also use the launcher (right-click on the tray icon) to select the appropriate screenshot type:
![image](https://github.com/flameshot-org/flameshot/assets/390889/ef6009db-e20c-4331-a857-553ed38d4c7b)
